python/paddle/fluid/dygraph/dygraph_to_static/print_transformer.py

```diff
@@ -39,45 +45,94 @@ def transform(self):
         self.visit(self.root)
 
     # NOTE: deal with print in PY3
-    def visit_Call(self, node):
-        assert isinstance(node, gast.Call)
-        if isinstance(node.func, gast.Name) and node.func.id == 'print':
-            var = self._get_print_var(node)
-            return self._construct_print_node(var)
+    def visit_Assign(self, node):
```

Comment:
This code contains bug, for example:

You had better add self.generic_visit for visiting the sons of Assign node. If print gast.Call node is not current node's value. Like:

`b = 1 or print("3")`

You could try, python is allowed to write above code.

Reply:
done, thx. for this case, keep `print` is `gast.Call `, do not replace by `gast.Assign`